api/v1alpha1/rsct_types.go

```diff
@@ -38,6 +38,7 @@ type RSCTSpec struct {
 type RSCTStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
+	CurrentStatus *string `json:"current_status,omitempty"`
```

Comment:
state will always be current here, can we rename to state here?

Reply:
sure, will make the change